src/forms/currentIncome.js

```diff
@@ -56,15 +56,15 @@ const IncomeForm = function ( props ) {
 
       <div style={{ textAlign: 'center' }}>
         <Header as='h4' textAlign='center'>
-          FOR A HOUSEHOLD SIZE OF <strong>{ otherProps.pageState.householdSize }</strong>:
+          FOR A HOUSEHOLD SIZE OF <strong>{ props.pageState.householdSize }</strong>:
```

Comment:
There's some inconsistency with `props.pageState` vs. `props.client` vs. `client = props.pageState`, etc. Is there a specific purpose behind it? If not, could it be made more consistent? We can possibly do that in a future iteration, though, if there isn't time for it now.

Reply:
`props.pageState` should never exist. All interaction with that should be passed through a `client` prop on the component.

Reply:
I mean that instead of having `props.pageState` *and* `props.client` *and* etc. we should just have one way of doing this. Maybe just send it in as 'client' from the very start. I just see the object being used here (in this PR) in different ways and I propose that it should be made more consistent.

Reply:
You're right. I think `props.pageState` shouldn't exist. Should be `props.client`. I forgot to change that. My mistake.